src/services/eventsService.js

```diff
@@ -10,24 +10,44 @@ async function registerEvent() {
             startDate: Date.now(),
             startTime: '23:59',
             endTime: '06:40'
-        }], // Eventually add another date!
+        }],
         imageUrl: 'https://cdn.akamai.steamstatic.com/steam/apps/1839940/capsule_616x353.jpg?t=1683123302',
         contacts: {
-            coordinates: { lat: '', long: '' },
+            coordinates: { lat: 'sda', long: 'sad' },
```

Comment:
These are not valid coordinates, if the front end tries to test visualisation of them, the code will break